docs/docs/0.3.0/policies/README.md

```diff
@@ -206,6 +209,14 @@ spec:
         service: redis
         version: '2.0'
 ```
+You also need to add labels to your services and use that label as a matching tag. Above, you  see that we utilize the `version` tag to help with canary deployment. Another common use-case is to add a `env` tag as you separate testing, staging, and production environments' services.
+
+#### Resolving Collision in Traffic Routes
+In the event where two traffic route many have colliding configurations, these are the expected behaviors:
```

Comment:
It worth explaining what does it mean for 2 configurations to collide. Maybe by giving an example

Reply:
expanded on collisions in all three examples with code snippets